addons/point_of_sale/static/src/xml/Screens/ReceiptScreen/OrderReceipt.xml

```diff
@@ -46,7 +46,7 @@
 
             <!-- Orderlines -->
 
-            <div class="orderlines">
+            <div class="orderlines" dir="ltr">
```

Comment:
small note: if this is used to force ltr direction, if it's possible, usually we would rather have the equivalent CSS (with directive telling RTLCSS to not inverse it):

```css
    direction: ltr/*rtl:ignore*/; /* rtlcss forced to keep ltr */
```